File: Project-Euler/Question-2/Even_Fibonacci_numbers.py
import time 
start = time.time()
million = 4000000
total = 0 
a = 1
b = 1

while a+b <= million:
    c = a + b
    if c % 2 == 0:
        total = total + c 
    b,a = c,b

elapsed = time.time() - start
print (total,elapsed)

File: Project-Euler/Question-2/Even_Fibonacci_numbers.cpp
#include <iostream>
#include <ctime>
#include <chrono>
using namespace std;
using namespace std::chrono;

int million = 4000000;
int total = 0 ;
int a = 1;
int b = 1;

int main()
{

    high_resolution_clock::time_point t1 = high_resolution_clock::now();
    while (a+b <= million)
    {
        int c = a + b;
        if (c % 2 == 0)
        {
            total = total + c;
        }
        b,a = c,b;
    }

    cout << total;
    cout << "\n";
    high_resolution_clock::time_point t2 = high_resolution_clock::now();

    auto duration = duration_cast<microseconds>( t2 - t1 ).count();

    cout << duration;
    cout << " microseconds";
    cout << "\n";
}